js/main.js

```diff
@@ -1,2 +1,37 @@
 /* global data */
 /* exported data */
+
+var $entryForm = document.querySelector('[data-view="entry-form"]');
+var $entryUrl = document.querySelector('#entry-url');
+var $entryTitle = document.querySelector('#entry-title');
+var $entryNotes = document.querySelector('#entry-notes');
+var $entryImage = document.querySelector('.entry-image > img');
+$entryUrl.addEventListener('blur', function (event) {
+  if (event.target.value) {
+    $entryImage.setAttribute('src', event.target.value);
+  }
+});
+
+$entryTitle.addEventListener('blur', function (event) {
+  if (event.target.value) {
+    $entryImage.setAttribute('alt', event.target.value);
+    $entryImage.setAttribute('title', event.target.value);
+  }
+});
+
+$entryForm.addEventListener('submit', function (event) {
+  event.preventDefault();
+  var entryFormObject = {
+    entryTitle: $entryTitle.value,
+    entryUrl: $entryUrl.value,
+    entryNotes: $entryNotes.value,
+    nextEntryId: data.nextEntryId,
+    entryId: data.nextEntryId - 1
```

Comment:
The `-1` shouldn't be necessary here 🤔 . Other wise, the first entry a user submits would have an id of `0` 😬

Reply:
Oh, yeah, I keep forgetting the users won't be dealing with "first index is actually at 0" like us